### PR TITLE
docs: redirects for removed pages

### PR DESCRIPTION
**Pull request by abdonpijpelink**
Reopening https://github.com/elastic/apm-server/pull/10891 to target `8.8`

### CODE REVIEW

**Review by simitt — APPROVED** [review]

**Review by simitt — CHANGES_REQUESTED** [review]
following redirects seem to be missing

```

INFO:build_docs:   - en/apm/guide/8.8/sourcemaps.html#sourcemap-rum-generate

INFO:build_docs:   - en/apm/guide/8.8/sourcemaps.html#sourcemap-rum-upload

INFO:build_docs:   - en/apm/guide/8.8/events-api.html#events-api-errors

INFO:build_docs:   - en/apm/guide/8.8/metadata-api.html#metadata-schema
```

File: docs/legacy/redirects.asciidoc [hunk 1]
```diff
@@ -326,3 +326,87 @@ Please see <<troubleshoot-apm>>.
...
+=== Errors
+
+This page has been deleted.
+Please see <<apm-overview>>.
+
+[role="exclude",id="sourcemaps"]
+=== How to apply source maps to error stack traces when using minified bundles
+
+This page has been deleted.
+Please see <<apm-overview>>.
```
Comment (colleenmcginnis) [inline docs/legacy/redirects.asciidoc:412]:
```suggestion
Please see <<apm-overview>>.

[role="exclude",id="sourcemap-rum-generate"]
=== Generate a source map

This page has been deleted.
Please see <<apm-overview>>.

[role="exclude",id="sourcemap-rum-upload"]
=== Upload the source map to Kibana

This page has been deleted.
Please see <<apm-overview>>.
```

File: docs/legacy/redirects.asciidoc [hunk 2]
```diff
@@ -326,3 +326,87 @@ Please see <<troubleshoot-apm>>.
 
 This page has moved.
 Please see <<configuring-howto-apm-server>>.
+
+[role="exclude",id="events-api"]
+=== Events Intake API
+
+This page has been deleted.
+Please see <<apm-overview>>.
+
```
Comment (colleenmcginnis) [inline docs/legacy/redirects.asciidoc:335]:
```suggestion

[role="exclude",id="events-api-errors"]
=== Errors

This page has been deleted.
Please see <<apm-overview>>.

```

File: docs/legacy/redirects.asciidoc [hunk 3]
```diff
@@ -326,3 +326,87 @@ Please see <<troubleshoot-apm>>.
...
+[role="exclude",id="metadata-api"]
+=== Metadata
+
+This page has been deleted.
+Please see <<apm-overview>>.
+
```
Comment (colleenmcginnis) [inline docs/legacy/redirects.asciidoc:347]:
```suggestion

[role="exclude",id="metadata-schema"]
=== Metadata Schema

This page has been deleted.
Please see <<apm-overview>>.

```

**Comment by abdonpijpelink** [comment]
@simitt It's passing all the checks now

**Review by simitt — APPROVED** [review]

**Review by bmorelli25 — APPROVED** [review]
Thanks everyone

**Comment by bmorelli25** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.9|[](https://github.com/elastic/apm-server/pull/11278)|
|✅|main|[](https://github.com/elastic/apm-server/pull/11279)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**